.gitignore

```diff
@@ -132,3 +132,5 @@ tmp_cache
 
 # python-language-server / Rope
 .ropeproject
+
+venv37/
```

Comment:
can you remove this unrelated change please

Reply:
Removed. Apologies, that snuck in.